NetTopologySuite.Samples.Console/Tests/Various/GraphBuilder.cs

```diff
@@ -1,137 +0,0 @@
-using System;

```

Comment:
Actually, files like this one had to be handled somehow, because the new SDK projects automatically include all files in the folder structure, whereas the old project files forced you to write out every single file that you wanted included.  This particular file (along with almost all of the other files that I outright deleted) was subtly omitted from the old monster .csproj files, so it just sat there doing nothing, not working.

So if I didn't delete this (and others), then I would instead have to do something like
```xml
<Compile Remove="Path\To\ThisFile.cs" />
```

Which seemed kinda silly.